### ISSUE TITLE
Bombing Run Multiball

### ISSUE THREAD

**Issue by theastropath** [theastropath/UT2K4CrowdControl#5]
It would be cool to have an effect that could spawn additional Bombing Run balls.

No idea what the consequences of just spawning one would be, if any (Does it mess up the compass?  or would it just be a silent bonus ball that would be hard to track?)

**Comment by Die4Ever** [comment]
my guess from out my ass is that the compass either has a variable saved for the (normally singular) ball, or the compass is using a foreach AllActors loop maybe with a break statement

I think in any of those cases the compass will still work, but likely only one of the balls will be tracked, but it is possible it could track them all

**Comment by theastropath** [comment]
Yeah, the GameInfo itself definitely tracks a single ball and that was my expectation, that it would purely be tracking that single ball.

That being said, I'm already overriding the HUDs for all game modes, so I might be able to make the compass show all the balls???
